services/web/server/tests/unit/with_dbs/test_registration.py

```diff
@@ -16,7 +16,7 @@
 EMAIL, PASSWORD = "tester@test.com", "password"
 
 
-async def test_regitration_availibility(client):
+async def test_regitration_availibility(loop, mock_orphaned_services, client):
```

Comment:
I see you had to put this fixture everywhere almost, is there no way of putting in the client fixture instead? so you do not have to change all these tests?

Reply:
It did not cross my mind to do it like that. I could try to do it and see what happens.